### ISSUE TITLE
[Feature request] Add support for environment management

### ISSUE THREAD

**Issue by olljanat** [microsoft/PowerShellForGitHub#342]
#### Feature Idea Summary
Afaiu this module does not currently support GitHub environments. There looks to be API support https://docs.github.com/en/rest/reference/repos#environments so it would be nice to get support to this one too to be able support use cases like having separate environment per pull request, etc.

**Comment by HowardWolosky** [comment]
Hi @olljanat,

It appears that they've restructured their API documentation.  Is this the Environment API support that you're referring to?

https://docs.github.com/en/rest/deployments/environments

**Comment by olljanat** [comment]
Yes and from UI point of view those should be visible on urls like github.com/org/repo/settings/environments.

That GitHub Enterprise feature for private repos but freely available for open source projects.

**Comment by HowardWolosky** [comment]
Great thanks.  Tagging to be tracked by the overall API completeness issue: #70
